tests/ios/xcodeproj/BUILD.bazel

```diff
@@ -4,6 +4,10 @@ xcodeproj(
     name = "Single-Static-Framework-Project",
     testonly = True,
     bazel_path = "bazelisk",
+    configs = [
+        "bar",
```

Comment:
nit: Does the bazel build work for these configs? It looks like `bar` and `foo` bazel configurations are not defined anywhere, so i would imagine `bazel build --configuration=bar` would fail. you probably don't need to fix this, but someone might see it in the future and be confused.

Reply:
You're right! Even though this is not being matched at the end it makes sense to be explicit about where those names should be coming from in a real scenario. Thx!